Source/UnitTests/Core/PatchAllowlistTest.cpp

```diff
@@ -136,3 +165,30 @@ TEST(PatchAllowlist, VerifyHashes)
                   << remaining_games.second.game_title;
   }
 }
+
+void CheckHash(const std::string& game_id, AllowList::iterator game_itr, const bool itr_end,
```

Comment:
A neater function interface would be:

```suggestion
void CheckHash(const std::string& game_id, GameHashes* game_hashes, const bool itr_end,
```